README.rst

```diff
@@ -41,6 +41,27 @@ If a Python core developer approved a PR made by anyone and added the "🤖 auto
 it will be automatically merged once all the CI checks pass.
 
 
+Setup Info
+==========
+
+.. highlight:: shell
+
+Requires Python 3.6+
+
+Create virtual environment and activate it::
```

Comment:
Since https://github.com/python/miss-islington/pull/620, we have a tox.ini config, so we can replace the venv setup, dependency installation and pytest call with installing and running tox.